spark-job-server-proposal.md

```diff
@@ -0,0 +1,61 @@
+# Adding Spark Job Server To Run Spark Jobs
+
+## Introduction
+
+This document provides design of spark job server which would trigger spark
+jobs either as scheduled or  batch jobs. Providing the user a ui for inputing
```

Comment:
We don't need to do that as we can create one of these: https://docs.openshift.com/container-platform/3.3/dev_guide/scheduled_jobs.html that would schedule the pod to be created